lib/ReactGridLayout.jsx

```diff
@@ -201,13 +258,20 @@ var ReactGridLayout = React.createClass({
     var activeDrag = {
       w: w, h: h, x: l.x, y: l.y, placeholder: true, i: i
     };
+
+    activeDrag = this.props.onResize(layout, layout, l, activeDrag, {e, element, size}) || activeDrag;
```

Comment:
In general I don't really like the assignment here; I don't see any demand for it and it is confusing - on dragStart and resizeStart, returning a value does nothing, on drag, you return the layout, on resize, you return the resizing item, and on dragStop and resizeStop you return the layout again.

Do you have a particular need for that functionality? Otherwise let's just remove the assignment altogether.